_posts/2016/2016-05-12-offsite-consulting.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: "Consulting offsite meeting"
+date: "2016-05-12 16:03:17"
+image: blog-header/blog-consulting-offsite-header.jpg
+categories: events
+authors: ["Pavlo Kalashnikov"]
+---
+
+On the 1st of Mai 2016 consulting team began its trip to Waldhotel in Bad Sulzburg. After a six-hour drive, we were greeted by nice staff of the hotel. The Hotel, which is located in the countryside, in a quiet location of the Schwarzwald. A perfect area for us to be clear about what went well and what could be done better in future. 
+
+{% image blog/blog-consulting-offsite-team-1.jpg %}{% endimage %}
+
+The purpose of this particular trip was to talk about opportunities and challenges faced by our team at the time and to envisage strategic plans for the future.
```

Comment:
Change to: 
...by our team over time as well as to come up with strategic plans for the future.